running/ingress-nginx.yaml

```diff
@@ -0,0 +1,663 @@
+# Includes everything to use nginx as an ingress
+# The NodePort service defined is coordinated with the HA Proxy setup by Continuu.
+# The node ports used by the NodePort service must not change.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ingress-nginx
+  labels:
+    app.kubernetes.io/name: ingress-nginx
+    app.kubernetes.io/instance: ingress-nginx
+
+---
+# Source: ingress-nginx/templates/controller-serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    helm.sh/chart: ingress-nginx-3.38.0
```

Comment:
Are we using Helm for this? Lots of Helm stuff in this file.

Reply:
I used the yaml they provide for bare metal installs. My guess is they use helm to generate the yaml for the different types of installs. There’s no actual helm stuff used at any point. I just didn’t feel like messing with any annotations or things, less I mess something up.